src/Adapter/Order/Delivery/SlipOptionsConfiguration.php

```diff
@@ -59,6 +60,7 @@ public function getConfiguration()
             'prefix' => $this->configuration->get(self::PREFIX),
             'number' => $this->configuration->getInt(self::NUMBER),
             'enable_product_image' => $this->configuration->getBoolean(self::ENABLE_PRODUCT_IMAGE),
+            'deliveryslip_language' => $this->configuration->getBoolean(self::DELIVERY_LANGUAGE),
```

Comment:
Just a detail ... is that name can be edited : "deliveryslip_language" ? or it's legacy and can cause a BC break ?
To respect the snake case convention it should be "delivery_slip_language".

Reply:
Nice, i updated the PR